test/image/strict-d3.js

```diff
@@ -2,6 +2,9 @@
  * strict-d3: wrap selection.style to prohibit specific incorrect style values
  * that are known to cause problems in IE (at least IE9)
  */
+
+// Do we need this process now that IE9 and IE10 are not supported?
```

Comment:
yes, if you look down below there's plenty here that applies to other browsers too. There may be some parts we could relax but no harm leaving it all for now.

Reply:
OK. I removed the comment in https://github.com/plotly/plotly.js/pull/5373/commits/9276d08e5298c0900daca36cfa9b150f57b400b5.